apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -196,16 +193,12 @@ class CPExpressInstallInstructions extends React.Component {
 
   render() {
     return (
-      <div id={installInstructionsId}>
-        <h2>{applabI18n.makerSetupCPXInstallHeader()}</h2>
+      <div className={styles.collapsedContent}>
+        <h4>{applabI18n.makerSetupCPXInstallHeader()}</h4>
```

Comment:
Nit: you could use `Typography`.

Reply:
Talked about this offline - Since the other headings and stuff on this page aren't using it, going to punt this one.